design/glossary.md

```diff
@@ -0,0 +1,16 @@
+| name | description |
+|---|---|
+| ipfs | interplanetary file system. hash-addressed storage in a peer-to-peer network. Content addressing is the big little idea behind IPFS. With content addressing (CIDs), you ask for a file using a hash of its contents. It doesn't matter where the file lives. Anyone in the network can serve that content.  |
+| ipld | interplanetary linked data. IPLD is a single namespace for all hash-inspired protocols. Through IPLD, links can be traversed across protocols, allowing you to explore data regardless of the underlying protocol. |
+| sphere | A Sphere is essentially a data structure for trivially associating names with content in a manner that is coherent and scales nicely with IPLD. |
+| petname | A petname system works like the address book on your phone. Instead of a global name system, you keep a personal address book that maps meaningful names to secure-decentralized addresses. Enables users to write human-readable names instead of DIDs when addressing other spheres. |
+| geist | German for ghost. robots that crawl the web of thoughts, mixing and combining them into new ones, surfacing relevant ideas.|
+| subtext | A plain text format, simpler than markdown. https://github.com/subconsciousnetwork/subtext |
+| supertext | essentially just a list of links to other content. It's a file format which, when rendered, consists entirely of transclusions. |
```

Comment:
"Supertext," is a working name only. It may be the final name, it may not be. I'm not sure what the best way to express this nuance should be. Any thoughts?